$NetBSD: patch-Makefile,v 1.1 2016/07/26 12:53:11 wiz Exp $

Fix include paths and link paths, and fix install target.

--- Makefile.orig	2016-07-26 03:17:27.000000000 +0000
+++ Makefile
@@ -1,12 +1,12 @@
 compiler = gcc
 target = odt2tex
-incdirs = -Iinc
+incdirs = -Iinc `pkg-config --cflags libzip`
 src = src
 rm = rm
 inc = inc
 optimize = -O2
 flags = -pipe -Wall $(optimize)
-libs = -lexpat -lzip
+libs = -lexpat `pkg-config --libs libzip`
 strip = -s
 
 # Objects for Main Binary
@@ -32,7 +32,8 @@ clean :
 	-@$(rm) $(target) $(src)/*.o 2>/dev/null
 
 install:
-	install -D $(target) $(DESTDIR)
+	install -d $(DESTDIR)$(PREFIX)/bin
+	install $(target) $(DESTDIR)$(PREFIX)/bin
 
 test:
 	./odt2tex in=testdoc.odt out=testdoc-out
